_posts/2018-08-24-running-strimzi-on-aks.md

```diff
@@ -0,0 +1,185 @@
+
+After downloading and installing the `kubectl` tool, you need to configure it with the right credentials for connecting to the just deployed Kubernetes cluster in AKS and the following command will do it.
+
+```
+az aks get-credentials --resource-group strimzigroup --name strimzicluster
+```
+
+To verify that the connection with the Kubernetes cluster is working you can just run the `kubectl get nodes` command for showing all the available nodes in the cluster.
+
+```
+NAME                       STATUS    ROLES     AGE       VERSION
+aks-nodepool1-24085136-0   Ready     agent     7m        v1.9.9
+aks-nodepool1-24085136-1   Ready     agent     7m        v1.9.9
+aks-nodepool1-24085136-2   Ready     agent     7m        v1.9.9
+```
+
+# Deploying Strimzi and an Apache Kafka
+
+Now that we have a Kubernetes cluster up and running in the cloud, we can deploy Strimzi in order to deploy an Apache Kafka cluster through the Cluster Operator.
+First of all you have to get the latest Strimzi release from [here](https://github.com/strimzi/strimzi-kafka-operator/releases).
+The first step is to modify the installation files accordingly to the namespace the Cluster Operator is going to be installed.
+
+```
+sed -i 's/namespace: .*/namespace: <my-namespace>/' examples/install/cluster-operator/*ClusterRoleBinding*.yaml
+```
+
+> If you want to deploy the Cluster Operator in the Kubernetes `default` namespace you have just to use `default` for the `<my-namespace>` placeholder in the above command.
+
+Deploying the Cluster Operator is just a matter of running:
+
+```
+kubectl create -f examples/install/cluster-operator
+```
+
+You can check that the related Pod is up and running just executing:
+
+```
+kubectl get pods
+NAME                                        READY     STATUS    RESTARTS   AGE
+strimzi-cluster-operator-65659b5f86-t5jsp   1/1       Running   0          1m
+```
+
+The Strimzi release provides an `examples` folder with examples YAML files for deploying an Kafka cluster with "ephemeral" and "persistent" storage and for deploying Kafka Connect as well.
+You can modify the related YAML files in order to customize for your needs but for the sake of this blog post, it's enough using the already provided Kafka cluster "ephemeral" example by running:
+
+```
+kubectl apply -f examples/kafka/kafka-ephemeral.yaml
+```
+
+For checking that the deployment is going on, you can run:
+
+```
+kubectl get pods -w
+```
+
+After few minutes, the Apache Kafka cluster will be running in the cloud!
+
+It's also possible to show the Kubernetes dashboard with all the Pods by running:
+
+```
+az aks browse --name strimzicluster --resource-group strimzigroup
+```
+
+It will open your default browser showing the dashboard as the following picture. 
+
+![Kubernetes dashboard with Strimzi and Apache Kafka running]({{ "/assets/2018-08-24-kubernetes-dashboard.png" | absolute_url }})
```

Comment:
You need to wrap it up better than this. A summary explaining what you did, emphasizing how easy+quick it was and alluding to the possibility for readers to now get on with writing/deploying their streaming apps.

Reply:
I added a possible conclusion to the blog post highlighting how AKS + Strimzi can help to set up a Kubernetes + Apache Kafka cluster in a few minutes.